cmd/loadTests.go

```diff
@@ -527,6 +530,10 @@ func setup(cmd *cobra.Command, args []string) {
 
 	logData.MaxTimeToSpinUpUsers = maxDurationFromArray(UserCreationTimeMaxPerThread).Seconds()
 
+	averageWaitTimeForPVCProvisioning := float64(0)
+	averageWaitTimeForPVCProvisioning = sumDurationFromArray(PipelineRunWaitTimeForPVCSumPerThread).Seconds() / float64(overallCount)
```

Comment:
Should we be dividing by `float64(overallCount)`, or by something like `totalCount(PipelineRunWaitTimeForPVCSumPerThread)`?